jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -233,7 +236,11 @@ function declare_xa_variables {
             if [[ $EJB_TIMER_DRIVER = *"mariadb"* ]]; then
                 dbType="mariadb"
             fi
-            eval ${prefix}_URL="jdbc:${dbType}://${host}:${port}/${database}"
+            if [[ $EJB_TIMER_DRIVER = *"mysql"* ]]; then
+                eval ${prefix}_URL="jdbc:${dbType}://${host}:${port}/${database}?enabledTLSProtocols=${MYSQL_ENABLED_TLS_PROTOCOLS:-TLSv1.2}"
```

Comment:
Is "enabledTLSProtocols" truly specific to the mysql driver? Do you know if it would work (and be good to have) with the mariadb driver?

Reply:
hmm that's a good question, I don't know if mariadb needs that, let me check the mariadb driver docs to see if I find something.

Reply:
Mariadb uses a different parameter name, which is enabledSslProtocolSuites

Reply:
conf applied to mariadb